### ISSUE TITLE
Bug for gathering college schedules in 2021

### ISSUE THREAD

**Issue by TravisRPetersen** [BillPetti/baseballr#225]
**Describe the bug**
When trying to run the get_ncaa_schedule_info function for teams with 2021 as the year, there is an error thrown that likely is due to a change in the format of the schedule on the ncaa website
**To Reproduce**
```
Error: Can't recycle `..1` (size 64) to match `..2` (size 54).
Traceback:

1. get_ncaa_schedule_info(teamid = 740, year = 2021)
2. game_result %>% dplyr::filter(.data$result != "Ppd" | .data$result != 
 .     "Canceled") %>% dplyr::bind_cols(game_info_url[, c(2, 3)]) %>% 
 .     dplyr::select(.data$row, .data$slug, .data$game_info_url)
3. dplyr::select(., .data$row, .data$slug, .data$game_info_url)
4. dplyr::bind_cols(., game_info_url[, c(2, 3)])
5. vec_cbind(!!!dots, .name_repair = .name_repair)

```

**Expected behavior**
Works for 2019 on all teams
team_schedule = get_ncaa_schedule_info(teamid=740, year=2019)

**Desktop (please complete the following information):**
Ran using Colab notebook 
Example here:
https://colab.research.google.com/drive/1TAQWhst0U_d4eC-BJ5xoasy0WyK4W0C-?usp=sharing

**Comment by ReedZahradnik5** [comment]
Hello,
Is there any update on this bug?
Thanks!

**Comment by saiemgilani** [comment]
This will be closed by pull request #229

**Comment by saiemgilani** [comment]
Just as a comment, as of version 1.0.2, this function is working altogether much better, worth updating from master
